### PR TITLE
Add Twitch Live Loadout Plugin

### PR DESCRIPTION
**Pull request by pepijnverburg**
This plugin allows the RuneLite client to synchronize game data to the Twitch Extension API (https://dev.twitch.tv/extensions) and give stream viewers live interfaces to interact. Currently inventory items, equipment items, bank items, skill levels and basic combat statistics are synchronized. 

It is built in such a way that any Twitch Extension can work with this (this is configurable via the plugin settings). The accompanied Twitch Extension is currently being reviewed by Twitch as well and is made completely in the style of the native game client (see screenshots in README).

Let me know if you have suggestions for improvements or would like to see a live stream with the plugin and Twitch extension in action. That is no problem to arrange (also doing the same with the Twitch reviewing team).

### CODE REVIEW

File: plugins/twitch-live-loadout [hunk 1]
```diff
@@ -0,0 +1,2 @@
+repository=https://github.com/pepijnverburg/osrs-runelite-twitch-live-loadout-plugin
```
Comment (dekvall) [inline plugins/twitch-live-loadout:1]:
```suggestion
repository=https://github.com/pepijnverburg/osrs-runelite-twitch-live-loadout-plugin.git
```

**Review by abextm — CHANGES_REQUESTED** [review]
Your support url is not a url

**Comment by pepijnverburg** [comment]
You mean the `support` property in https://github.com/pepijnverburg/osrs-runelite-twitch-live-loadout-plugin/blob/ffe082ebb7f87bde83204620ca6fb934d2ce6518/runelite-plugin.properties#L3? Was expecting a way of contacting the maintainer there. I'll change it to the Github page.

**Comment by abextm** [comment]
your build is still failing

**Pull request merged**